### PR TITLE
Doc/tutorial update

### PR DESCRIPTION
**Pull request by D-GopalKrishna**
Closes - None (A doc tutorial update)

Implemented solution: Changed the doc to include or change
- Changed the namespace name error 
- Added a package-lock.json possible error
- Added a WSL2 reference link to stackoverflow question

How to test this PR: Read the file changed i.e. at path - /docs/tutorials/simple-date-clock-application.adoc

### Sanity checks:
- [ ] The pull request is explicitly linked to the relevant issue(s)
- [ ] The issue is well described: clearly states the problem and the general proposed solution(s)
- [ ] From the issue and the current PR it is explicitly stated how to test the current change
- [ ] The labels in the issue set the scope and the type of issue (bug, feature, etc.)
- [ ] All the automated test checks are passing
- [ ] All the linked issues are included in one milestone
- [ ] All the linked issues are in the Review/QA column of the [board](https://app.zenhub.com/workspaces/cloud-harness-5fdb203b7e195b0015a273d7/board)
- [ ] All the linked issues are assigned

### Breaking changes (select one):
- [x] The present changes do not change the preexisting api in any way
- [ ] This PR and the issue are tagged as a `breaking-change`

### Possible deployment updates issues (select one):
- [x] There is no reason why deployments based on CloudHarness may break after the current update
- [ ] This PR and the issue are tagged as `alert:deployment`

### Test coverage (select one):
- [ ] Tests for the relevant cases are included in this pr
- [x] The changes included in this pr are out of the current test coverage scope

### Documentation (select one):
- [x] The documentation has been updated to match the current changes
- [ ] The changes included in this PR are out of the current documentation scope

### Nice to have (if relevant):
- [ ] Screenshots of the changes
- [ ] Explanatory video/animated gif

### CODE REVIEW

File: docs/tutorials/simple-date-clock-application.adoc [hunk 1]
```diff
@@ -71,9 +71,15 @@ If everything is good, you're in the right path to use {ch}!
 
 == How to setup your local cluster using minikube (if not done yet)
 
+
 {ch} is designed to help you generate all the required artifacts for a deployment on a Kubernetes cluster.
 For a local deployment, you can use minikube, which is basically https://minikube.sigs.k8s.io/docs/start/[a small {kub} cluster on your machine].
 
+
+[IMPORTANT - WSL2]
+If following this tutorial in a Windows WSL2 - then check https://stackoverflowteams.com/c/metacell/questions/84/[this] for help!
```
Comment (filippomc) [inline docs/tutorials/simple-date-clock-application.adoc:80]: It's better not to point to the stackoverflow page as it's not visible to the public. CloudHarness is open source instead. Can create a specific readme here and link to that

Comment (D-GopalKrishna) [reply docs/tutorials/simple-date-clock-application.adoc:80]: @gidili suggestion sounds good. I'll do that!

Comment (gidili) [reply docs/tutorials/simple-date-clock-application.adoc:80]: @D-GopalKrishna I was joking, no need to do that 😆

**Comment by D-GopalKrishna** [comment]
@filippomc Updated the link with internal relative path in adoc file.

**Review by filippomc — APPROVED** [review]
Nice, thanks @D-GopalKrishna and welcome as a contributor!

**Pull request merged**